assets/js/components/user-input/CancelUserInputButton.js

```diff
@@ -30,7 +35,7 @@ import { CORE_LOCATION } from '../../googlesitekit/datastore/location/constants'
 import Link from '../Link';
 const { useSelect, useDispatch } = Data;
 
-export default function CancelUserInputButton() {
+export default function CancelUserInputButton( { disabled = false } ) {
```

Comment:
The default state for the `disabled` property is `undefined`. Let's stick with it:
```suggestion
export default function CancelUserInputButton( { disabled } ) {
```